go.mod

```diff
@@ -1,12 +1,14 @@
 module github.com/docker/buildx
 
-go 1.22.0
+go 1.23
+
+toolchain go1.23.7
 
 require (
 	github.com/Masterminds/semver/v3 v3.2.1
 	github.com/Microsoft/go-winio v0.6.2
 	github.com/aws/aws-sdk-go-v2/config v1.27.27
-	github.com/compose-spec/compose-go/v2 v2.4.8
+	github.com/compose-spec/compose-go/v2 v2.4.9
```

Comment:
I think PR title is wrong and should be `bump compose-go to v2.4.9` :sweat_smile: